uri-scheme.md

```diff
@@ -0,0 +1,43 @@
+# URI Scheme
+## Scope
+This URI scheme describes the wormhole file transfer application. For other use cases of the wormhole spec and URI scheme has not been defined yet.
+
+The application ID for this protocol is always assumed to be `lothar.com/wormhole/text-or-file-xfer`
+
+## Description
+
+The `wormhole-transfer` URI Scheme is used to encode a wormhole code for file transfer as a URI. This can then be used to generate QR codes, or be opened by the platform URI handler to open a supporting client.
+
+The general format looks like this, and assumes default values for all query fields:
+`wormhole-transfer:{code}`
+
+`{code}` is the [URL / percent encoded](https://en.wikipedia.org/wiki/Percent-encoding) wormhole code. While common codes may not require any encoding, it must be made sure that percent-encoding and decoding is applied to support all possibilities.
+
+It can be extended by appending a [query string](https://en.wikipedia.org/wiki/Query_string).
+
+## Query fields
+Applications MUST parse all query fields specified below and fail if they contain unknown fields or unsupported parameter values. Query values are percent-encoded.
+
+* `version`: The version of the URI scheme.  At the moment, only version 0 is specified. Clients must check this for compatibility.
+Default: `0`
+
+* `rendezvous`: The rendezvous server and protocol to use, including its port. The versioned endpoint is supposed to be added by the client implementation, eg. `/v1`
+Default: `ws://relay.magic-wormhole.io:4000`
+
+* `role`: The type of operation requested. Valid values are:
+		* `follower`: The URI *generating* client is asking to lead the transit connection (and most likely send a file) and is requesting the URI *parsing* client to follow the leader (and most likely receive a file).
```

Comment:
Style nit: other parts of the documentation use 2 or 4 spaces for indentation

Reply:
Whoops, that was a mistake :D